asteroid/losses/mse.py

```diff
@@ -25,6 +25,10 @@ class PairwiseMSE(_Loss):
     """
 
     def forward(self, est_targets, targets):
+        if targets.size() != est_targets.size() or targets.ndim != 3:
+            raise TypeError(
+                f"Inputs must be of shape [batch, n_src, *], got {targets.size()} and {est_targets.size()} instead"
+            )
```

Comment:
The first part of the check is fine but the second one isn't. 
Those forward are made to work for any input shape. 
Sorry if you lost time on the checks and the tests.

Reply:
Ah, right. But for *SDR we always have `2 <= ndim <= 3`?

Reply:
Or shall accept any shape for `time`, i.e. `(batch, n_src, time1, time2, ...)`?

Reply:
In *SDR, the checks are correct. We can remove them in the MSE.